src/main/java/seedu/address/model/assignment/Assignment.java

```diff
@@ -1,17 +1,16 @@
 package seedu.address.model.assignment;
 
-import seedu.address.commons.util.ToStringBuilder;
-import seedu.address.model.person.Name;
-import seedu.address.model.tag.Tag;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
 
-import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.Objects;
 import java.util.Set;
 
-import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.person.Name;
```

Comment:
Use `seedu.address.model.assignment.Name` instead

Reply:
I just removed it, not needed since its the same package